packages/http/test/uri-template.test.ts

```diff
@@ -0,0 +1,33 @@
+import { describe, expect, it } from "vitest";
+import { parseUriTemplate } from "../src/uri-template.js";
+
+it("no parameter", () => {
+  expect(parseUriTemplate("/foo").parameters).toEqual([]);
```

Comment:
does that mean we don't need path property anymore? because all paths need to be uri-template style. `/foo` is just a simplest uri template.

Reply:
Yes `@path` and `@query` can be redundant, they still have use if you want to use a different name or have those props nested somewhere where they wouldn;'t get auto discovered. And yeah if you define them all in the uri template then there is not a real difference between query or path param they are all params to build the uri.